docker-compose/docker-compose.yml

```diff
@@ -52,25 +52,12 @@ services:
 
     transform-router:
         mem_limit: 512m
-        image: quay.io/alfresco/alfresco-transform-router:1.1.1
+        image: quay.io/alfresco/alfresco-transform-router:1.2.0-A2
         environment:
           JAVA_OPTS: " -Xms256m -Xmx512m"
           ACTIVEMQ_URL: "nio://activemq:61616"
 
-          IMAGEMAGICK_URL: "http://transform-core-aio:8090"
-          IMAGEMAGICK_QUEUE : "org.alfresco.transform.engine.aio.acs"
-
-          PDF_RENDERER_URL : "http://transform-core-aio:8090"
-          PDF_RENDERER_QUEUE : "org.alfresco.transform.engine.aio.acs"
-
-          LIBREOFFICE_URL : "http://transform-core-aio:8090"
-          LIBREOFFICE_QUEUE : "org.alfresco.transform.engine.aio.acs"
-
-          TIKA_URL : "http://transform-core-aio:8090"
-          TIKA_QUEUE : "org.alfresco.transform.engine.aio.acs"
-
-          MISC_URL : "http://transform-core-aio:8090"
-          MISC_QUEUE : "org.alfresco.transform.engine.aio.acs"
+          CORE_AIO_URL : "http://transform-core-aio:8090"
 
```

Comment:
Should the CORE_AIO_QUEUE variable be included here?

Reply:
No, it's not needed.

As with previous releases (when using individual x5 T-Engines) it's already pre-configured with a default to match Core AIO T-Engine default :-)